RIN.WebAPI/DB/DB.Zone.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System.Security.AccessControl;
+using System.Threading.Tasks;
+using Dapper;
+using Microsoft.Extensions.Logging;
+using ProtoBuf;
+using RIN.WebAPI.Models.ClientApi;
+using RIN.WebAPI.Models.Common;
+using RIN.WebAPI.Models.DB;
+using RIN.WebAPI.Utils;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace RIN.WebAPI.DB
+{
+    public partial class DB
+    {
+        public async Task<List<ZoneSettingsResp>> GetZoneSettings()
+        {
+            const string SELECT_SQL = @"SELECT 
+                    id,
+                    zone_id,
+                    mission_id,
+                    gametype,
+                    instance_type_pool,
+                    is_preview_zone,
+                    displayed_name,
+                    displayed_desc,
+                    description,
+                    displayed_gametype,
+                    cert_required,
+                    xp_bonus,
+                    sort_order,
+                    rotation_priority,
+                    skip_matchmaking,
+                    queueing_enabled,
+                    team_count,
+                    min_players_per_team,
+                    max_players_per_team,
+                    min_players_accept_per_team,
+                    challenge_enabled,
+                    challenge_min_players_per_team,
+                    challenge_max_players_per_team,
+                    images
+                    FROM webapi.""ZoneSettings""
+                    WHERE is_active = True";
+
+            var zone_settings = await DBCall(async conn => conn.Query<DBZoneSettings>(SELECT_SQL));
+
+            var zones = new List<ZoneSettingsResp>(zone_settings.Count());
```

Comment:
No need for the new list or foreach loop.

``var zone_settings = await DBCall(async conn => conn.Query<DBZoneSettings>(SELECT_SQL));``
Returns a list we can just use.

For the db names not matching you can use 
```SQL
SELECT 
id AS Id,
zone_id AS ZoneID,
mission_id AS MissionID,
gametype AS GameType,
```

etc